docs/scripts/scan_examples.c

```diff
@@ -81,9 +90,13 @@ int main(int argc, char* argv[])
    lookup = calloc(argc, sizeof(lookup_t));
    for (j = 0; j < argc; ++j) {
       lookup[j].index = j;
+      int len = strlen(argv[j]);
```

Comment:
`lookup[j].filename = strdup(argv[j]);` ?

Reply:
Can't you just store the `argv[i]` pointer, no need to make a copy of the string it points to?

Reply:
@SiegeLord That seems very reasonable yes - and I have tested your method too now @pedro-w - that seems even better. I'll push to my branch here using this instead. Thanks!